user_templates/variables.cfg

```diff
@@ -73,6 +73,8 @@ variable_disable_motors_in_end_print: False
 ## Automatically turn-off heaters in the END_PRINT macro
 variable_turn_off_heaters_in_end_print: True
 
+## Automatically disable motors on timeout
+variable_disable_motors_on_timeout: False
```

Comment:
Can you update the default to `True` here? The goal is to align it with the `default(1)` from the macro